sql/hive/src/main/scala/org/apache/spark/sql/hive/HiveStrategies.scala

```diff
@@ -244,6 +244,10 @@ private[hive] trait HiveStrategies {
             ExecutedCommand(RunnableDescribeCommand(planLater(o), describe.output)) :: Nil
         }
 
+      case drop: DropTable =>
+        // TODO: hive0.14 support drop temp table
+        ExecutedCommand(DropHiveTable(drop.tableName, drop.isExists, drop.temporary)) :: Nil
+
```

Comment:
Can you please also add a test case where we drop a temporary table from a hive context?